index.html

```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    <title>Pick a name, any name!</title>
+  </head>
+  <body>
+    <h1>Welcome!</h1>
+    <h2>Regarding the program found here:</h2>
+    <p>Hello and welcome to this lovely little web page! I'm glad you've stopped by to test your knowledge of your classmates! In toay's scripting, we'll be testing how well you can guess the middle names of your fellow students!</p>
+
+    <p>In this handy dandy little site, you'll be asked to pick a co-educatee's first name and then guess their middle name! You'll have unlimited guesses and hints along the way to help out! Don't worry if you don't guessing correctly, this whole thing is only for my grade and in no way, shape, or form reflects on you as a human being and will NOT mean I fail and then lose my house and have to pimp out my kitty for a living. Really, no biggie!
+  <hr />
+  <h2>This is completely unrelated to the program</h2>
+    <p>Now, because I'm bored and I have nothing else to put into this space, I'm going to use some of the common tags used in HTML! For example, wrapping a word in a tag with a b makes it look <b>bold</b>. Or you can use the i key for <i>italics</i>. Additionally, the <strong>strong</strong> and <em>em (emphasis)</em> might LOOK like they're <b>bold</b> or <i>italicized</i>, but they really aren't!</p>
```

Comment:
Bold and Italicize tags should not be used here. Those are style tags and will be managed by your CSS.